docs/overview.md

```diff
@@ -100,7 +100,124 @@ source on your build, if you wish.
 <link href="node_modules/megadraft/dist/css/megadraft.css" rel="stylesheet">
 ```
 
+### Custom Toolbar
 
+You can provide custom actions using MegadraftEditor `actions` property. Megadraft Toolbar provides [custom actions](https://github.com/globocom/megadraft/blob/master/src/actions/default.js) like **Bold** and *Italic*, additionaly you can provide your own custom actions or even add some custom actions to the default ones.
+
+Example: Add an underlined action
+
+```js
+import React from "react";
+import ReactDOM from "react-dom";
+import {MegadraftEditor, editorStateFromRaw} from "megadraft";
+import CustomSidebar from 'my/sidebar/path';
+
+import actions from "megadraft/actions/default";
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {editorState: editorStateFromRaw(null)};
+    this.onChange = ::this.onChange;
+  }
+
+  onChange(editorState) {
+    this.setState({editorState});
+  }
+  render() {
+    const custom_actions = actions.concat([{type: "inline", label: "U", style: "UNDERLINE", icon: Underlined}]);
+    return (
+      <MegadraftEditor
+        editorState={this.state.editorState}
+        onChange={this.onChange}
+        actions={custom_actions}/>
+    )
+  }
+}
+
+/**
+ * Underlined Icon.
+ * svg extracted from [Material UI underlined icon](https://material.io/icons/#ic_format_underlined)
+ */
+class Underlined extends React.Component {
```

Comment:
Not exactly a problem, but how about removing the icon here?

I believe it's not required for the example.